### DISCUSSION TITLE
v2导入v1设置后出现两个影响使用的bug

### DISCUSSION THREAD

**Discussion by oscarwty** [the1812/Bilibili-Evolved#2460] [提问 / Q&A] [answered]
firefox93+tampermonkey，彻底删除v1，v2先下载了全部组件插件合集，再导入v1设置
现在鼠标移动到左侧，却从右侧显示设置按钮，导致无法打开设置窗口
另外，在打开firefox左侧书签栏的情况下，鼠标移动到哪都无法打开设置页面
![Snipaste_2021-11-01_21-46-53](https://user-images.githubusercontent.com/6859653/139681974-6d7f8dea-98f4-46a0-9f1a-4e35352f796b.png)

**Comment by the1812** [comment, marked as answer]
> v2先下载了全部组件插件合集

重新试一遍, 不要下全部组件

**Reply by oscarwty** [reply]
感谢，已解决。有个小bug是面板在右侧的时候，下载插件界面，鼠标放上去插件的提示会被设置窗口挡住。
